bundles/org.openhab.binding.atlona/src/main/java/org/openhab/binding/atlona/internal/pro3/AtlonaPro3PortocolHandler.java

```diff
@@ -192,7 +201,7 @@ String login() throws Exception {
         try {
             response = callback.getResponse();
             if (!response.equals("")) {
-                logger.info("Altona protocol violation - didn't start with an inital empty response: '{}'", response);
+                logger.debug("Altona protocol violation - didn't start with an inital empty response: '{}'", response);
             }
         } catch (Exception e) {
```

Comment:
Please do not catch `Exception`. I assume `IOException` is enough. Catching `Exception` also catches NPE and hides programming errors.

Reply:
I completely agree. However in this instance, the exception being caught is explicitly thrown as Exception in getResponse() on lines 1260 & 1262. I could change those to be IOException or a custom exception. What do you think?

Reply:
I just looked at the rest of the code. This should never have been merged. Removing the `Exception` here will require some bigger refactoring, which is clearly out of scope here. Leave it as is.